nkms/characters.py

```diff
@@ -254,9 +268,29 @@ class Ursula(Character):
     interface = None
     interface_ttl = 0
 
+    def __init__(self, urulsas_keystore=None, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        if urulsas_keystore:
+            self.keystore = urulsas_keystore
+        else:
```

Comment:
For an Ursula, this might be very dangerous to start a non persistent storage system. Maybe we should not let her store kFrags if this is the case, otherwise she's doomed to lose them. Maybe fail instead of warn for Ursula?

Reply:
I mostly agree.  For now, though, this makes it so much easier to test.  Maybe have an affirmative kwarg like, `use_memory_db` or something?

Reply:
Or what if we make it impossible to use a memory db unless we replace it by mocking or in a conftest?

Idk if it's a good practice